src/services/auth.services.js

```diff
@@ -0,0 +1,62 @@
+const { ApiError } = require('../exeptions/auth.error');
+const { User } = require('../models/user');
+const { hashPassword } = require('../utils/password/hashPassword');
+const { randomUUID } = require('crypto');
+
+const signUp = async ({ email, password }) => {
+  const hashedPassword = await hashPassword(password);
+
+  const user = await User.create({
+    email,
+    password: hashedPassword,
+    activationToken: randomUUID(),
+  });
+
+  return user;
```

Comment:
Verify that the `signUp` function returns all necessary user details, such as the activation token, if required for further processing in the signup flow.